fe1-web/src/resources/strings.ts

```diff
@@ -141,7 +141,8 @@ const STRINGS = {
   election_voting_method: 'Voting Method',
   election_method_Plurality: 'Plurality',
   election_method_Approval: 'Approval',
-  election_version_identifier: '1.0.0',
+  election_version_open_ballot: 'open-ballot',
+  election_version_secret_ballot: 'secret-ballot',
```

Comment:
was there a change in the specification ? It'd make sense, I must've missed it 😅

Reply:
Yes that is one of the changes of #918. Since we use the version field to differentiate between open and secret ballot elections and given there are no other "versions" at the moment I thought we might as well chose names instead of numbers?

https://github.com/dedis/popstellar/blob/work-fe1-evoting-request-election-key/protocol/query/method/message/data/dataSetupElection.json#L13-L16

Reply:
there might be multiple versions of the secret ballot, but it's no different than any other aspect of the communication protocol in the application. For now, we've just ignored the problem of protocol versioning and assumed that everyone would be running the same version of the software, which is Good Enough (TM) for now 😄 
If and when we'll need to address that, we'll have to use a more global solution anyways.

Reply:
Hmm yes that is a good point as well.. I saw that the evoting document mentions semantic versioning 🤔 So maybe use the version field as before and add another type field?